### PR TITLE
Moving symbol ')' to next line

### PR DESCRIPTION
**Pull request by MaximRybak**
This PR is ready for review.
Summary
This PR provides new features:
- Moving symbol ')' to next line;

CLA
☑︎ I have signed the CLA

### CODE REVIEW

**Review by AByzhynar — APPROVED** [review]

**Comment by AByzhynar** [comment]
@LuxoftAKutsan Please review

**Comment by vilyukh** [comment]
Change target branch to develop

**Review by BSolonenko — APPROVED** [review]

**Comment by dkandyba** [comment]
@MaximRybak This feature already exists. To turn on, you should run **cmake-format** with follow argument: _--dangle-parens 1_ or add in config-file as _dangle_parens: True_

File: cmake_format/formatter.py [hunk 1]
```diff
@@ -1206,7 +1207,7 @@ def assert_lt(self, cursor):
 
   def forge_cursor(self, cursor):
     self._cursor = cursor
-
+
```
Comment (LuxoftAKutsan) [inline cmake_format/formatter.py:1210]: @MaximRybak I suppose that formating existing python code is not a good idea.
I can see it not the first time. Please take care about changes in your commits.

**Review by LuxoftAKutsan — APPROVED** [review]

**Pull request closed without merging**